airflow/contrib/operators/druid_operator.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+from airflow.exceptions import AirflowException
+from airflow.hooks.druid_hook import DruidHook
+from airflow.models import BaseOperator
+
+
+class DruidOperator(BaseOperator):
+    """
+    Allows to submit a task directly to druid
+
+    :param conn_id: the druid ingest connection id
+    :type conn_id: str
+    :param json_index_spec: The json formatted string that contains the index specification
+    :type json_index_spec: str
+    """
+
+    def __init__(
+        self,
+        json_index_spec=None,
+        json_index_file=None,
+        conn_id='druid_ingest_default',
```

Comment:
Ok, since the `query_ingest_conn_id` was not used anymore, I changed it to `conn_id`. But I revoked the changes if you prefer `druid_ingest_conn_id`. I don't have a strong opinion about this.